test/e2e/template/helmfile/testdata/snapshot/release_template_inheritance/output.yaml

```diff
@@ -33,13 +30,4 @@ data:
   inheritedBaseLabel: base
   template2: template2
   template2Label: template2
-  templates:
```

Comment:
yaml.v2 encoded this in the flow style but we are not interested in a specific yaml library to encode the same go map to a yaml dict in flow style or non-flow style. I just removed this key out of values to make the output look the same between yaml.v2 and goccy/go-yaml. See input.yaml for the change.